wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: wiki
+title: "Jamulus Walk-Through"
+lang: "en"
+permalink: "/wiki/Walk-Through"
+---
+# Jamulus walk-through
+Let's have a look at what Jamulus looks like and what you should do after having installed the software.
+## First start
+Before you start Jamulus:
+1. Close all other applications
+2. Plugin everything (Ethernet cable, audio gear, etc.)
+
+... to minimise CPU load and to ensure that no other application tries to use your sound card.
+
+You can have a look at [Hardware Setup](Hardware-Setup) for more information on how to set up your hardware. If you need help setting up ASIO4All on Windows, have a look at the [ASIO4All Setup guide on the Windows installation page](Installation-for-Windows#setting-up-asio4all).
+
+### Jamulus main window
+If you open Jamulus, you will see a window which looks like this:
+
+![Jamulus main window](https://user-images.githubusercontent.com/20726856/100796017-4cfa4500-3420-11eb-9d35-aa206d392a5a.png)
+
+### 1. Setting up a profile
+To allow others to identify who you are, you should set your profile. To do so, click on "View" (bar on the top)>"My Profile..."
+Now you will see this:
+
+![Musician Profile](https://user-images.githubusercontent.com/20726856/100796411-e590c500-3420-11eb-9b62-e50d02cb009b.png)
+
+Fill at least "Alias/Name" to set your name and close the window.
+
+### 2. Connecting to a server
+Now you can connect to a server. Click on "Connect" in the Jamulus main window, select a genre list (selection box on the top left), select a server, and click on "Connect". Now you should be connected to the server. If you have problems here, they are most likely due to wrongly configured audio gear or network issues. Have a look at the [Troubleshooting page](Client-Troubleshooting).
+### The first Jam
+Congrats! You can now start jamming with Jamulus. The sliders you see on the right work as a mixing desk: everything you change here will change what you hear. If you move a slider down, this user will be quieter, if you move it up, this user will be louder for you. It's that easy! Everything else like "PAN", "SOLO", "GRP", ... are advanced options. You can try them of course!
```

Comment:
"work as a mixing desk" :-) OK, but for the record I think a large number of problems with the use of Jamulus stem from people taking this too literally. In fact your next sentence shows this. If it **did** behave like a mixing desk then it would be "everything you change here will change _what everyone hears_".  Jamulus does not behave like a mixing desk. It only looks like one. 

Also, I would avoid possibly diminishing statements like "It's that easy!" because if somebody doesn't in fact find it easy, they'll feel stupid. 

BTW for "PAN", "SOLO", "GRP" we should probably refer the reader to the software manual.

Reply:
To be discussed...

What's your proposal? We need to clarify that it's the own "mixing desk" of course.

Reply:
I think if we keep with a looser mixing desk analogy, then how about:

"The sliders you see on the right are your own personal mix. Everything you change here will change what you hear, but won't affect others. If you move a slider down, that user will be quieter, if you move it up, that user will be louder for you."

In fact we use "personal mix" in the UI I think.

Reply:
Yes. Agree. Fixed